build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -25,9 +25,14 @@
         "dear_imgui::impl_sdl", "dear_imgui::impl_vulkan"
     ],
     # LLVM
+    "@llvm-project//llvm:asm_parser": ["LLVMAsmParser"],
     "@llvm-project//llvm:core": ["LLVMCore"],
+    "@llvm-project//llvm:core": ["LLVMCore"],
```

Comment:
`LLVMCore` looks like a duplicate you might want to drop.

Reply:
There's a quirk in our source code migration setup right now that is causing this. @GMNGeoffrey and @asaadaldien are figuring out a fix. Thanks for noticing :)